components/form/overview.md

```diff
@@ -12,22 +12,12 @@ position: 0
 
 The <a href = "https://www.telerik.com/blazor-ui/form" target="_blank">Form for Blazor</a> allows you to generate a form based on your model and to manage customized forms. You can control the component through various parameters, achieve the desired layout by using the [default editor](#automatic-generation-of-fields) or add [custom ones]({%slug form-formitems%}), set the [orientation]({%slug form-orientation%}) and organize those editors in [groups]({%slug form-formgroups%}) and [columns]({%slug form-columns%}). 
 
-#### This article is separated in the following sections:
 
-
-* [Use the Telerik Form for Blazor with a model](#use-the-telerik-form-for-blazor-with-a-model)
-* [Use the Telerik Form for Blazor with an EditContext](#use-the-telerik-form-for-blazor-with-an-editcontext)
-* [Component Reference](#component-reference)
-* [Automatic Generation of fields](#automatic-generation-of-fields)
-* [Data Annotation Attributes](#data-annotation-attributes)
-* [Features](#features)
-
-
-## Use the Telerik Form for Blazor With a Model
+## Creating Blazor Form Bound to a Model
```

Comment:
The two "Creating" sections are 99% identical and take up a lot of space. Consider one of the following options - 
* Merge the two sections and examples into one.
* Use only the model approach and afterwards mention that EditContext usage is similar. Optionally, provide a short non-runnable snippet to avoid any possible confusion.